content/docs/TechnicalWritingCourse/information_gathering.md

```diff
@@ -64,11 +64,14 @@ This involves answering the question: "***What do I aim to achieve with my techn
 
 **Here are some steps to take to define the purpose of your document:**
 
-1. **Identify the Goal:** Identify the primary goal of your document. Are you informing, instructing, or persuading your audience? Your goal will determine the structure and tone of your document.
-2. **Define The Outcome:** After reading your document, what should your user be able to do? Would you want them to be able to implement a specific Technology? Or would you want them to understand a complex concept? Or do you want them to be able to make an informed decision based on the information gained from your document?
-3. **Content Scope:** With your goal and outcome clearly defined, you can draft out the scope of your document's content based on the technical background of your audience. What information should your audience have already, and what information do you want them to gain from your document?
+##### 1. Identify the Goal:
+Identify the primary goal of your document. Are you informing, instructing, or persuading your audience? Your goal will determine the structure and tone of your document.
+##### 2. Define The Outcome:
+After reading your document, what should your user be able to do? Would you want them to be able to implement a specific Technology? Or would you want them to understand a complex concept? Or do you want them to be able to make an informed decision based on the information gained from your document?
+##### 3. Content Scope:
+With your goal and outcome clearly defined, you can draft out the scope of your document's content based on the technical background of your audience. What information should your audience have already, and what information do you want them to gain from your document?
 
-According to Google’s Technical Writing course:
+##### According to Google’s Technical Writing course:
```

Comment:
Please put this in a callout using the syntax I shared on the Whatsapp group.

Reply:
This should be the description

`A Good documentation = knowledge and skills your audience needs to do a task − your audience's current knowledge and skills`